README.txt

```diff
@@ -6,7 +6,8 @@ Introduction
 
 ``asn1ate`` is a Python library for translating ASN.1 into other forms.
 It is intended for code generation from formal ASN.1 definitions, and a
-code generator for ``pyasn1`` is included.
+code generator for ``pyasn1`` is included.  Additional code for the
```

Comment:
Let's make this: 

    [...] from formal ASN.1 definitions, and code generators for ``pyasn1`` as well as ``Quick DER`` are included.

Readers of this don't care in which order things were added.